packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -221,7 +221,7 @@ function VM-Install-Raw-GitHub-Repo {
         # $powershellCommand = "Get-Content README.md"
         # $powershellCommand = "Import-Module module.ps1; Get-Help Main-Function"
         [Parameter(Mandatory=$false)]
-        [string] $powershellCommand
+        [string] $powershellCommand = ""
```

Comment:
why is this needed?

Reply:
I thought that was what led to: https://github.com/mandiant/VM-Packages/actions/runs/8651261154/job/23721615509#step:4:460

```
[adconnectdump.vm] vm.common.psm1 [+] ERROR : A parameter cannot be found that matches parameter name 'powershellCommand'.
At C:\ProgramData\_VM\vm.common\vm.common.psm1:228 char:185
+ ... utableName $executableName -withoutBinFile -powershellCommand $powers ...
+                                                ~~~~~~~~~~~~~~~~~~
ERROR: A parameter cannot be found that matches parameter name 'powershellCommand'.
The upgrade of adconnectdump.vm was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\adconnectdump.vm\tools\chocolateyinstall.ps1'.
```

But now I'm realizing that it was just due to not having the `common.vm` version set properly in a dependency.


We do the same for `$executableName`: https://github.com/mandiant/VM-Packages/pull/987/files#diff-42c583682dc7cee57079eab399b94fbe5a60b1fcec5626a25a7092e197cb7308L217